pyproject.toml

```diff
@@ -26,7 +26,7 @@ include = ["zzz_poetry_dynamic_versioning.pth"]
 python = "^3.5"
 dunamai = "^1.5"
 tomlkit = ">= 0.4"
-jinja2 = "^2.11.1"
+jinja2 = {version="^2.11.1", optional=true}
```

Comment:
> May be easier to just broaden the jinja version spec

Yeah, I'd rather do that for now, so that we don't have to change the install instructions now and then change them again in the near future for the official plugin system. I think we could do this:

```toml
jinja2 = [
    { version = "^2.11.1", python = "~3.5" },
    { version = "^3.0.0", python = "^3.6" }
]
```